app/retail/templates/settings/account.html

```diff
@@ -51,7 +51,21 @@
     border: 0.2rem solid #1AB56D;
   }
 </style>
-<form id="settings" method="POST" class="col-md-8 m-auto">
+<form id="settings" method="POST" action="{% url 'account_settings' %}" class="col-md-8 m-auto">
+  <div class="form-group mb-5">
+    <h5>{% trans "Account type" %}</h5>
+    <p>{% trans "Let us know so we could optimize the best experience for you!" %}</p>
```

Comment:
Text suggestion: "Let us know your account type so we can optimize your experience"

Reply:
This is from the design I agree with you I will just let @PixelantDesign know about this change ^